test/unit/dom.js

```diff
@@ -95,4 +95,46 @@ describe('DOM', function () {
       expect(helpers.fixture().querySelector('.my-circle').skated).to.equal(true);
     });
   });
+
+  describe('Document Fragments', function () {
+    var frag;
+    var created;
+    var attached;
+    var MyEl;
+
+    beforeEach(function () {
+      created = false;
+      attached = false;
+      MyEl = skate('my-element', {
```

Comment:
`0.12.x` doesn't have the safe tag name stuff, so just reusing `'my-element'` — I guess the tests here don't have a problem with registering the same skate `id` more than once?